docs/architecture.md

```diff
@@ -34,4 +34,4 @@ In addition, this resource offer process repeats when tasks finish and new resou
 
 While the thin interface provided by Mesos allows it to scale and allows the frameworks to evolve independently, one question remains: how can the constraints of a framework be satisfied without Mesos knowing about these constraints? For example, how can a framework achieve data locality without Mesos knowing which nodes store the data required by the framework? Mesos answers these questions by simply giving frameworks the ability to **reject** offers. A framework will reject the offers that do not satisfy its constraints and accept the ones that do.  In particular, we have found that a simple policy called delay scheduling, in which frameworks wait for a limited time to acquire nodes storing the input data, yields nearly optimal data locality.
 
-You can also read much more about the Mesos architecture in this [technical paper](http://mesos.berkeley.edu/mesos_tech_report.pdf).
+You can also read much more about the Mesos architecture in this [technical paper](http://mesos.apache.org/assets/papers/nsdi_mesos.pdf).
```

Comment:
I think you can use "https://www.usenix.org/legacy/events/nsdi11/tech/full_papers/Hindman.pdf"

Reply:
https://www.usenix.org/conference/nsdi11/mesos-platform-fine-grained-resource-sharing-data-center is even better because it has links to video, audio, slides and the paper.

Reply:
I [amended my commit](https://github.com/apache/mesos/pull/247/commits/46e9e4221f705f94dbe8120c498153dc3af714da) to use the URL you suggested.

Do you think it's still worth pinging someone about maybe fixing the broken URL [referenced by external websites](https://www.google.com/search?q=%22mesos.berkeley.edu/mesos_tech_report.pdf%22&oq=%22mesos.berkeley.edu/mesos_tech_report.pdf%22)?